types/mapkit.Map.d.ts

```diff
@@ -106,6 +107,31 @@ declare namespace mapkit {
      * Changes the map's visible map rectangle to the specified map rectangle.
      */
     setVisibleMapRectAnimated(mapRect: mapkit.MapRect, animate?: boolean): this;
+    /**
+     * Sets a constraint for the center of the map.
+     */
+    cameraBoundray: CameraBoundaryDescription;
+    /**
+     * Sets a constraint for the center of the map.
+     */
+    setCameraBoundaryAnimated(coordinate: mapkit.CoordinateRegion, animate?: boolean): this;
+    // TODO: setCameraBoundaryAnimated(mapRect: mapRect, animate?: boolean): this;
```

Comment:
I'm not sure about this one. Is overloading working for declaration files? Would it be fine setting `coordinate: mapkit.CoordinateRegion | mapkit.MapRect`? This won't match the parameter name defined in the Apple docs...